release.json

```diff
@@ -22,114 +22,24 @@
         "WINDOWS_DDNPM_SHASUM": "6b5a1b1fb5aad64a23c119501cddfc16a98a6d1ad5b3f8cc91a603284627af19",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
-    "6.23.0-rc.6": {
-        "INTEGRATIONS_CORE_VERSION": "7.23.0-rc.4",
-        "OMNIBUS_SOFTWARE_VERSION": "7.23.0-rc.1",
-        "OMNIBUS_RUBY_VERSION": "7.23.0-rc.1",
+    "6.23.0": {
+        "INTEGRATIONS_CORE_VERSION": "6.23.0",
+        "OMNIBUS_SOFTWARE_VERSION": "6.23.0",
+        "OMNIBUS_RUBY_VERSION": "6.23.0",
```

Comment:
I think you'll want to use the `7.23.0` tags instead (based on what we did for the 6.22.0 release)

Reply:
Why is that?